cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectCluster.java

```diff
@@ -237,10 +236,7 @@ protected static <C extends KafkaConnectCluster> C fromSpec(Reconciliation recon
             result.configuration = config;
         }
         if (result.tracing != null)   {
-            if (JaegerTracing.TYPE_JAEGER.equals(result.tracing.getType())) {
-                config.setConfigOption("consumer.interceptor.classes", JaegerTracing.CONSUMER_INTERCEPTOR_CLASS_NAME);
-                config.setConfigOption("producer.interceptor.classes", JaegerTracing.PRODUCER_INTERCEPTOR_CLASS_NAME);
-            } else if (OpenTelemetryTracing.TYPE_OPENTELEMETRY.equals(result.tracing.getType())) {
```

Comment:
Should we keep this and throw a warning here that `type: jaeger` is not supported and will be ignored? Same for the other model classes as well.

Reply:
Wdym by "throw a warning"? Thorwing an exception and failing the riconcile or just logging a warning? My feeling is the second one which is more likely something like "ignoring" but asking just to confirm.

Reply:
Log a warning, not throw ... sorry.

Reply:
I am going to have something like this ...

```java
if (JaegerTracing.TYPE_JAEGER.equals(result.tracing.getType())) {
                LOGGER.warnCr(reconciliation, "Tracing type \"{}\" is not supported anymore and will be ignored", JaegerTracing.TYPE_JAEGER);
            }
```

Reply:
Yeah, that looks fine. I guess you can hardcode the type into the log message maybe as it is always the same ... but it is fine even as it is 👍